rmgpy/rmg/model.py

```diff
@@ -1591,6 +1591,14 @@ def addReactionLibraryToEdge(self, reactionLibrary):
                 # Don't add to the edge library reactions that were already processed
                 self.addReactionToEdge(rxn)
 
+        if self.saveEdgeSpecies:
+            from rmgpy.chemkin import markDuplicateReaction
+            newEdgeReactions = self.edge.reactions[numOldEdgeReactions:]
+            checkedReactions = self.core.reactions + self.edge.reactions[:numOldEdgeReactions]
+            for rxn in newEdgeReactions:
+                markDuplicateReaction(rxn, checkedReactions)
```

Comment:
Could you remind me why we don't need to `checkedReactions()` when adding a seed library to core?

Reply:
In `addSeedMechanismToCore`, we call `self.markChemkinDuplicate` which calls `Chemkin.markDuplicateReactions` to process all reactions and mark duplicates. Here, we assume that the existing reactions have already been marked properly, so we just check the new reactions against the previously checked ones.